sync-core/src/main/java/com/cloudant/sync/replication/BasicPushStrategy.java

```diff
@@ -312,14 +317,25 @@ private ItemsToPush missingRevisionsToJsonDocs(
         return allOpenRevisions;
     }
 
+    public String getReplicationId() {
+        HashMap<String, String> dict = new HashMap<String, String>();
+        dict.put("source", this.sourceDb.getIdentifier());
```

Comment:
Fixed that - in pull it now uses the getIdentifier() convenience method which is equivalent